Cargo.toml

```diff
@@ -13,16 +13,17 @@ version = "2.1.0"
 include = ["src/**/*", "LICENSE", "README.md"]
 
 [dependencies]
-fnv = "1.0.6"
-itertools = "0.8.1"
+fnv = "1.0.7"
+itertools = {version = "0.10.5", optional = true}
 lazy_static = "1.4.0"
-regex = { version = "1.3.1", optional = true }
+regex = { version = "1.7.0", optional = true }
 
 [dev-dependencies]
 speculate = "0.1.2"
 
 [features]
 enable_regex = ["regex"]
+tokenize = ["itertools"]
 default = []
```

Comment:
tokenize should be a default feature

Reply:
I personally use this only to convert kana and I think there lots of users of wanakana use it for this purpose only as well. But this also wasn't intended to be added to this pr.

Reply:
I have no problem to add this to the lib. Is there a problem with the itertools dependency? (no_std support?)

Reply:
I just prefer to have as less dependencies as possible in my code or better said I see it as a good practice to keep them as few as possible and I don't need `tokenize` in my current project.